src/main/java/nl/tudelft/contextproject/model/level/util/CorridorExponentialExtension.java

```diff
@@ -0,0 +1,269 @@
+package nl.tudelft.contextproject.model.level.util;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import nl.tudelft.contextproject.util.RandomUtil;
+
+/**
+ * Utility class that contains methods which expand corridors using an exponential distribution.
+ */
+public final class CorridorExponentialExtension {
+	
+	/**
+	 * Minimum extension of corridors.
+	 */
+	public static final int MINIMUM_EXTENSION = 0;
+	
+	/**
+	 * Maximum extension of corridors.
+	 */
+	public static final int MAXIMUM_EXTENSION = 2;
+	
+	/**
+	 * Extra corridor extensions for the corridor corners.
+	 */
+	public static final int EXTRA_CORNER_EXTENSION = 1;
```

Comment:
This comment doesn't really add anything as the variable name is already descriptive enough.